statsmodels/genmod/families/family.py

```diff
@@ -345,7 +366,9 @@ def deviance(self, endog, mu, freq_weights=1., scale=1.):
         -----
         If a constant term is included it is defined as
 
-        :math:`deviance = 2*\\sum_{i}(Y*\\log(Y/\\mu))`
+        .. math::
+
+           D = 2 * \sum_i (Y_i * freq\_weights_i * \log(Y_i / \mu_i))/ \phi
```

Comment:
style: freq_weights should be in front so we have the standard formula (without freq weights after it all together)
`D = 2 * \sum_i (freq\_weights_i * Y_i * \log(Y_i / \mu_i))/ \phi`